app/controllers/application_controller.rb

```diff
@@ -55,6 +55,12 @@ def deny_if_not_admin
     raise ActiveRecord::RecordNotFound
   end
 
+  def prevent_action_if_registration_closed
```

Comment:
Since this method is only used in the registrations controller it should me moved there.